sopel/trigger.py

```diff
@@ -286,6 +286,22 @@ class Trigger(unicode):
     ``PRIVMSG``. Other event types like ``NOTICE``, ``NICK``, ``TOPIC``,
     ``KICK``, etc. must be requested using :func:`.plugin.event`.
     """
+    ctcp = property(lambda self: self.tags.get('intent', None))
+    """The CTCP command (if any).
+
+    :type: str
+
+    Common CTCP commands are ``ACTION``, ``VERSION``, and ``TIME``. Other
+    commands include ``USERINFO``, ``PING``, and they are used for ``DCC``.
+
+    .. versionadded:: 7.1
+
+    .. note::
+
+        This attribute must be used instead of the ``intent`` tag that will be
+        removed in a future version of Sopel, as it represents a feature in
+        IRCv3 spec that is long dead.
```

Comment:
```suggestion
    .. important::

        Use this attribute instead of the ``intent`` tag in :attr:`tag <tags>`.
        Message intents never made it past the IRCv3 draft stage, and Sopel will
        drop support for them in a future release.
```

I'm suggesting approximately the fifth version of this docstring. It took a ton of self-restraint to skip mentioning "Sopel should have already removed these ages ago". 😂